src/components/People/Facts.stories.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import Facts from './Facts'
+
+export default {
+  title: 'component/Facts',
+  component: Facts,
+}
+
+const Template = args => <Facts {...args} />
```

Comment:
you dont need the args here: `const Template = () => <Facts />`
You only need the args if you have some variables/props that you need for the component to be displayed in storybook